pkg/networkmanager/interfaces.js

```diff
@@ -1686,7 +1686,7 @@
             if (member_settings.connection.group == group_settings.connection.uuid ||
                 member_settings.connection.group == group_settings.connection.id ||
                 member_settings.connection.group == group_iface)
-                return cockpit.resolve();
+                return Promise.resolve();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18487-20230310-143650-f841503b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)